src/test/java/com/training/assignments/StudentTest.java

```diff
@@ -0,0 +1,23 @@
+package com.training.assignments;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class StudentTest {
+
+    @Test
+    public void testStudent()
+    {
+        Student student=new Student();
+        student.setName("Naviya");
```

Comment:
As Naviya used twice ,Use variable like String name=Naviya and use it.
same for Lastname ,RollNo & SchoolName.